src/Features/CSharp/Portable/Completion/KeywordRecommenders/AbstractNativeIntegerKeywordRecommender.cs

```diff
@@ -19,6 +19,11 @@ internal abstract class AbstractNativeIntegerKeywordRecommender : IKeywordRecomm
 
         private static bool IsValidContext(CSharpSyntaxContext context)
         {
+            if (context.IsPreProcessorDirectiveContext)
+            {
+                return false;
+            }
+
             if (context.IsStatementContext ||
                 context.IsGlobalStatementContext ||
                 context.IsPossibleTupleContext ||
```

Comment:
Why there is a difference between regular & scripting modes in this code path?

Reply:
Right, I have to retract the part about script mode (that's just where I encountered it). It affects regular mode too if the file is empty for example. The faulty check was in `SyntaxTreeExtensions.IsGlobalStatementContext`, which returned `true`.